### PR TITLE
Module cleanup: Removed silly assignments, unreachable code and more

### PR DESCRIPTION
**Pull request by andygrunwald**
This PR doesn't contain functionality changes.
It is only about code cleanup:

* Removed silly assignments (where both sides are the same)
* void return values used
* Removed unnecessary javascript label
* Removed unreachable statement

### CODE REVIEW

File: modules/server/ping_server.inc.php [hunk 1]
```diff
@@ -20,7 +20,7 @@ function ping_server($host, $port)
         $available = $success;
 
         $special_info = "";
-        // Weitere Daten f�r FTPs herrausfinden
+        // Weitere Daten f�r FTPs herrausfinden
```
Comment (M4LuZ) [inline modules/server/ping_server.inc.php]: Encoding strikes again ;)

File: modules/server/ping_server.inc.php [hunk 2]
```diff
@@ -85,7 +85,7 @@ function ping_server($host, $port)
         } // END: If Type=FTP
 
 
-        // Weitere Daten f�r IRCs herrausfinden
+        // Weitere Daten f�r IRCs herrausfinden
```
Comment (M4LuZ) [inline modules/server/ping_server.inc.php]: dito

**Comment by andygrunwald** [comment]
@M4LuZ Gimme a second.

**Comment by andygrunwald** [comment]
@M4LuZ Fixed.

**Pull request merged**